pkgs/development/web/minify/update-go-version-mod.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/go.mod b/go.mod
+index cebe363..f9ae9c8 100644
+--- a/go.mod
++++ b/go.mod
+@@ -1,6 +1,6 @@
+ module github.com/tdewolff/minify/v2
+ 
+-go 1.13
++go 1.18
```

Comment:
Would you mind trying to upstream this?

Reply:
Done: https://github.com/tdewolff/minify/pull/593

Reply:
It's now upstream, I don't know when will be the next release so maybe it could be merge so the main modifications exists for the next release.